protocols/luci-proto-yggdrasil/htdocs/luci-static/resources/protocol/yggdrasil.js

```diff
@@ -257,6 +293,86 @@ return network.registerProtocol('yggdrasil',
 			o=ss.option(form.Value,"password",_("Password"));
 			o.optional=true;
 
+			// Jumper tab
+			try {
+				s.tab('jumper',_('Jumper'));
+			} catch(e) {};
+
+			o=s.taboption(
+				'jumper',
+				form.HiddenValue,
+				'hidden_value',
+				_(' '),
+				_('<a href="https://github.com/one-d-wide/yggdrasil-jumper">Yggdrasil Jumper</a> is independent project that aims to transparently reduce latency of a connection over Yggdrasil network, utilizing NAT traversal to bypass intermediary nodes. It periodically probes for active sessions and automatically establishes direct peerings over internet with remote nodes running Yggdrasil Jumper without requiring firewall or port configuration.')
```

Comment:
Please revise to e.g.
```javascript
				_('%s is independent project that aims to transparently reduce latency of a connection over Yggdrasil network, utilizing NAT traversal to bypass intermediary nodes. It periodically probes for active sessions and automatically establishes direct peerings over internet with remote nodes running Yggdrasil Jumper without requiring firewall or port configuration.').format('<a href="https://github.com/one-d-wide/yggdrasil-jumper">Yggdrasil Jumper</a>');
```

Reply:
Would be nice to break up the really long texts (to encourage i18n). 

So:
```js
_('sentence 1.') + ' ' +
_('sentence 2.') + ' ' +
```

Reply:
Please don't use `_(' ')` in those couple of locations.  valuable space gets used up in i18n files with 'space' string.

Just `+ ' ' +` will do.